### DISCUSSION TITLE
Can we concatenate different types of rooms to be a large room?

### DISCUSSION THREAD

**Discussion by liuxz-cs** [allenai/ai2thor#802] [Q&A] [answered]
Excuse me, I want to know if I can concatenate different types of rooms to form a large room ? For example, we concantenate rooms from Kitchen, Bathroom, Living room and Bedroom respectively to form a large house consisting of 4 rooms ? If ai2thor supports this function, how can we do this?

**Comment by winthos** [comment, marked as answer]
AI2THOR currently does not innately support this function. However, this is possible using some combination of manipulation from the Unity Editor.

Additionally, there has been some work in this area by others using our framework. Please take a look at this paper for further information: http://avn.csail.mit.edu/

**Comment by liuxz-cs** [comment]
> AI2THOR currently does not innately support this function. However, this is possible using some combination of manipulation from the Unity Editor.
> 
> Additionally, there has been some work in this area by others using our framework. Please take a look at this paper for further information: http://avn.csail.mit.edu/

Okay, thanks a lot.
